### PR TITLE
[monitoring][rabbitmq] update dashboard and alerts

### PR DESCRIPTION
**Pull request by clsv**
## Description
update of rmq grafana dashboards and prometheus alerts

## Why do we need it, and what problem does it solve?
old rmq alerts don't work because they use old metrics

## Changelog entries
```changes
section: monitoring-applications
type: feature
summary: Rabbitmq dashboard and alert improvements
```

### CODE REVIEW

File: ee/fe/modules/340-monitoring-applications/applications/rabbitmq/prometheus-rules/rabbitmq.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - name: applications.rabbitmq
   rules:
   - alert: ApplicationRabbitMQMemoryUsage
-    expr: rabbitmq_node_mem_used{job="rabbitmq"} / rabbitmq_node_mem_limit{job="rabbitmq"} * 100 > 85
+    expr: rabbitmq_process_resident_memory_bytes{job="rabbitmq"} / rabbitmq_resident_memory_limit_bytes{job="rabbitmq"} * 100 > 85
```
Comment (vladimirGuryanov) [inline ee/fe/modules/340-monitoring-applications/applications/rabbitmq/prometheus-rules/rabbitmq.yaml:4]: Your changes will break alerts and dashboards if someone uses older versions of RMQ metrics. I think it's better to use PromQL like this:
```
((rabbitmq_node_mem_used{job="rabbitmq"} / rabbitmq_node_mem_limit{job="rabbitmq"}) or (rabbitmq_process_resident_memory_bytes{job="rabbitmq"} / rabbitmq_resident_memory_limit_bytes{job="rabbitmq"})) * 100 > 85
```
This way, we don't break backward compatibility with older versions RMQ metrics.  This also applies to all subsequent changes.

File: ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json [hunk 2]
```diff
@@ -58,27 +74,30 @@
...
+        "textMode": "auto"
       },
-      "pluginVersion": "7.0.6",
+      "pluginVersion": "8.5.13",
       "targets": [
         {
-          "expr": "sum(rabbitmq_queue_messages_ready * on(instance) group_left(rabbitmq_cluster) rabbitmq_identity_info{namespace=\"$namespace\",job=\"rabbitmq\"})",
+          "datasource": {
+            "type": "prometheus",
+            "uid": "P0D6E4079E36703EB"
```
Comment (vladimirGuryanov) [inline ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json:98]: Need to use the variable, like:  "$ds_prometheus"

File: ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json [hunk 3]
```diff
@@ -88,42 +107,44 @@
...
       "title": "Ready messages",
-      "transparent": true,
       "type": "stat"
     },
     {
-      "cacheTimeout": null,
-      "datasource": "main",
+      "datasource": {
+        "type": "prometheus",
+        "uid": "P0D6E4079E36703EB"
```
Comment (vladimirGuryanov) [inline ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json:116]: Need to use the variable, like:  "$ds_prometheus"

File: ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json [hunk 4]
```diff
@@ -139,27 +160,30 @@
...
+        "textMode": "auto"
       },
-      "pluginVersion": "7.0.6",
+      "pluginVersion": "8.5.13",
       "targets": [
         {
-          "expr": "sum(rate(rabbitmq_channel_messages_published_total[60s]) * on(instance) group_left(namespace) rabbitmq_identity_info{namespace=\"$namespace\",job=\"rabbitmq\"})",
+          "datasource": {
+            "type": "prometheus",
+            "uid": "P0D6E4079E36703EB"
```
Comment (vladimirGuryanov) [inline ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json:184]: Need to use the variable, like:  "$ds_prometheus"

File: ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json [hunk 5]
```diff
@@ -5540,121 +4373,164 @@
...
   "time": {
-    "from": "now-3h",
+    "from": "now-1h",
     "to": "now"
   },
   "timepicker": {
-    "refresh_intervals": ["15s", "30s", "1m", "5m", "10m"],
-    "time_options": ["5m", "15m", "1h", "6h", "12h", "24h", "2d", "7d", "30d"]
+    "refresh_intervals": [
+      "15s",
```
Comment (vladimirGuryanov) [inline ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json:4513]: We have a 30-second scrape interval as usual, so a fast refresh doesn't make sense.

File: ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json [hunk 6]
```diff
@@ -219,71 +245,75 @@
...
           "values": false
-        }
+        },
+        "textMode": "auto"
       },
-      "pluginVersion": "7.0.6",
+      "pluginVersion": "8.5.13",
       "targets": [
         {
-          "expr": "count(rabbitmq_channel_messages_published_total * on(instance) group_left(rabbitmq_cluster) rabbitmq_identity_info{namespace=\"$namespace\",job=\"rabbitmq\"})",
```
Comment (vladimirGuryanov) [inline ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json:242]: In this case, we need to use 'or' for backwards compatibility with older versions. This also applies to all subsequent changes.

**Review by vladimirGuryanov — CHANGES_REQUESTED** [review]

File: ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json (hunk 5 above)
Comment (clsv) [reply ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json]: Changed the refresh intervals as in the other dashboards.

**Review by nabokihms — APPROVED** [review]

**Pull request merged**